src/chromo/common.py

```diff
@@ -95,6 +95,12 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self == other
 
+    def __getitem__(self, item):
+        return getattr(self, item)
+
+    def __setitem__(self, item, value):
+        setattr(self, item, value)
```

Comment:
That is still bad, the Zen of Python says: "There should be one (obvious) way to do it". Adding two ways to access the fields of a dataclass `foo.x` and `foo["x"]` is going against that principle. It is also superfluous.

Reply:
If we want CrossSectionData to act more like an array of values, we should inherit from `typing.NamedTuple`, which provides that for free.

Reply:
How to iterate over fields in dataclass?

Reply:
> However, I think we should just implement the method _mul_radd and be done with it. 

Ok